reference-artifacts/iam-policies/key-policy-basic.txt

```diff
@@ -6,10 +6,10 @@
             "Sid": "Enable IAM User Permissions",
             "Effect": "Allow",
             "Principal": {
-                "AWS": "arn:aws:iam::863254859172:root"
+                "AWS": "arn:aws:iam::444455556666:root"
```

Comment:
role-trust-policy.txt uses `xxACCTNUMBERxx` as the placeholder. Would make sense to align these